config/manifests/bases/network-observability-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,59 @@
ICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAKPD94cGFja2V0IGVuZD0idyI/PoZeawcAAAAGYktHRAD/AP8A/6C9p5MAAAAJcEhZcwAACxMAAAsTAQCanBgAAAAHdElNRQflDA4PNilvgPSHAAAHDUlEQVRYw82XS2xcVxnHf+fcxzzuTMbO+JnETtMY8rCJEEUOiC7ooqKIBVKkLppFEAuQUIKQ2GTVLmDFhg01kSohWKCqUlRUKYtWLCpIiQg1QUkbm7zsPMb2xK+xPXPndWbOOSzmkRnbKSndcKSje3XOvd/3P9/zfwSfY/w0kRAGxoEXgBFgsLm1DGSAaxJmfhOG9lllimf56GwicRT4CfAqMPxfPs8CF4ELU2F46wsBOJtI7Ad+BZxufTs0MsJQXx+9ySS+5+FIiQEWVle5ef165+8WeBs4PxWGi58bwNlE4lXgLaAnGgQcHRvj4MAAQTSK4ziNKWX7vVit8rebN3lxYoKlXI6r168T5nIAm8CPp8Lw4jMDOJtIvA78AuDw2BhfPnCAqOftqrj9vm2tpjX/vHePv1+50hL7xlQY/nK7Ludpyh3X5asTE4z09SGlBNGBVYidyEX3ius4jPT3c2BkhHuZDLpWe2nS9820UpefCqBp9t86rsuJY8foCYIdgnmKYqU1a6USBVXDdx1cIRBCkAoCDo2OcuvhwxaI2WmlZncAaAbc+0D0yOHDpOJxykoxc/cuG8Ui6VQKsc0KAHVjuLK6xoexOJUjRzETXyE7PEx2+TEDWuNISRCNMjw4yOzt2wAvT/r+H6eVKnQBmPT9C8DkwOAg/akUAOv5PBsbG5RLJdLpNJ7T7TFjLe9mFlka2sf+kVGG9w2x78B+jh4/zsQrr/CPhQwDuRxSCFKJBMTjLGQyUWBwWqk/AciOPD/teh79qRRaa7TWpIKAVE8Pg0ND+I5DXWvqxlDXmqJSvHtvntvIpjEs1oI1FmMMUkpOnjrFp/k82hisMXx9bIzE3r0Ap5s6GxaY9P3XgW/0pdME0eiTAJGS3mSS1LZYWKtU+cPdOT4phMQTAYlEgngQEIvHiUQjOI5DTSkqlQpza2uMFAogBI6U+LEY8/fvC8BMK/WBcy6ZFMDvgORAOt3wc4cy0aworeetrTwfLGb5/uh+ruc2Ea5LNBYlEonieS5SSrTWlMtlCvk8D7KPGSvkG5nR18feQ4f4+KOPAEZPRiK/dq2148BwJBIBQGvdrGO2K1VqxnA1t0nS8/jB8weJui6DvsdSGJLfyhOLx/E8D2uhXC4TbVqymM2iTUNm32uvER8aYvSdd3g0NzdsrR13m40Fz/Mayu3OPrJVq3EjH/K13h76Y1GEtdSNYSKV5P7yGhu5HK7rYq1FVasUCnl838da2JtdQvelG03i4kX83l6GBwZ4NDcH8ILb7Go4TdN1AbCWTKVKDXhxbw+e4zyxEHBszx4uPV6lkM8DllpNUSoWicfjeL6P2trk2z17qBuDBcoPHqAePSIVj7dEjLitlmqa0W2bvq5by7I2DPg+Sd/Dao3usIq2lr8ur/Kd/jSz+QKZrTzVapVCoUAkEiViND8a2Ud0l4N1VJNBt53TxrRPVzUWJQz7fA8H215vgTPWcmV1nYOxGOO9Kb7Zn2ahVGZdKYpa8+elJX4+foR0JLJrTBljnpTsJploA9CAlJbkNnN3Crm2mWckGuH5RBytNY7j8Fwi4LBM8pflFU7tH2Y0CBoW3SWmVK3Wel2WTSbTLjJWa6SUxMfH2wWpPY3hZj5k0Hc5EPHba639hbDIcrnCZH+665+61l1zo1BoAchI4For/VrzzKVL/Oy99zh+7lzXj/dLZfpcyYDrdinWWlOt1/nw8QrfHR7EGrNjvxPs8tpaC8A1VwoxY6zNGmOG602T+7EYQkq8IGi7YV0bko7DHiGeBKu17bj4ZHOLb6V7iQqxq99bo6IUy4uLAFkJM87HSjHp+6PASWstQgg+vXyZ2zduMHvhAgChsUSFIJBiBy8A2KzVKNQ1Y4lg1/3OyH+4usrKygrA798Mw/dbveABcNZaKwBq6+tszswgAAW4QhB5imAD/LtY5sSeBMYYiuUyUkocKXfwhrox/GtmBl2rWeCH00qtOQDTSq1N+v6XgBPW2i4FUoidtKnjRJmq4rlYBEcIMtksGxsb5La26Ekmd4CYy2ZZa5z+7akwfKuVhq1xHvge0LM9/fQuvsRaSsYSEwLXWqpKUVOqYRWtCSsVPPeJ+FwYcn9+vkVSz+9gRNNKFSZ9f77J/bvzdzdaJgShMfS6Trt1V2s1tNa4nkdfT0+bQYWVCrN37mAbBejMVBhe3ZUTTis1O+n7BniJjijfrlgAZWtJyCY5bSoKYjGCIKA3lWqbf6tU4s78PKZh1TemwvDCZ7LiaaUubwex23CapLMbm8BxHASgjWF5c5OFxcWWjGej5R0gZoGXgai1FmvtkybyGbRcG8NWqcTSygrFMGz5/Mz2k//PV7PWKR0pkc0ppMQaQ61eRzUD8Qtfzf4vLqfbx7lkUjQp3FOv50KImTcLhWe+nv8HSg24inLC1x4AAAAASUVORK5CYII=
```

Comment:
as a provisional image, I used this: 
![image](https://user-images.githubusercontent.com/939550/146036578-772cb31b-2fae-4fcb-9786-9df22425f632.png)

Reply:
cc @jhadvig ;-)